packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/index.js

```diff
@@ -39,6 +38,8 @@ const useLazyComponents = (componentUids = []) => {
       setStore(Object.fromEntries(componentStore));
     };
 
+    const newUids = componentUids.filter((uid) => !componentStore.get(uid));
```

Comment:
Could you not set the variable without useMemo the same way you did before? Pass it to use state and then use it here in the useEffect. Maybe I'm missing something here but not sure I understand why we need to do the filtering twice.

Reply:
@not-vainglorious could you please add this back, based on this feedback https://github.com/strapi/strapi/pull/15623#discussion_r1097628989 it looks like you had it right :). Please just add a comment explaining why it is there.